### PR TITLE
Implement the scale config setting for registers; update tests

### PR DESCRIPTION
**Pull request by ch-evergen**
## What?

* Add a MessageTransformer class which applies the `scale` setting from config for holding registers
* Add tests to test scaling up and down for ints and floats, including the following scenarios:
 ** make sure ints stay as ints
 ** re-validate after scaling so we catch it if if value is scaled up beyond datatype boundary

  
## Why?

Communication between different systems requires that command values can be scaled.

## Testing/Proof

Automated tests

## Open Source Reminder

Please remember that this repository is open source. Be careful to avoid including any sensitive information in your changes or your comments. If you need to discuss something sensitive, please do so through a private channel.

### CODE REVIEW

File: app/message.py [hunk 1]
```diff
@@ -59,6 +64,22 @@ def validate(cls, input_type, value):
...
+        # TODO validate register inputs
+
+
+class MessageTransformer:
+    @classmethod
+    def transform(cls, configuration, value):
+        tvalue = value
+        if configuration.scale:
+            value = configuration.scale * value
+            if "INT" in configuration.data_type:
```
Comment (serroba) [inline app/message.py:76]: can you explain why is this needed?
Why do we care about int but not about floats?

Comment (ch-evergen) [reply app/message.py:76]: It seems like `value` is automatically a float after the multiplication operation, so if it's meant to be an int we need to turn it back into one.

**Review by serroba — APPROVED** [review]
